newsletter.css

```diff
@@ -73,58 +107,46 @@
     width: 450px;
 }
 
-.sk-body {
-    margin: 0;
-    padding: 0;
-    background-color: white;
+.sk-img-right {
+    padding-left: 90px;
 }
 
-.sk-wrapper {
-    width: 100%;
-    table-layout: fixed;
-    background-color: #0C2B35;
-}
-.sk-webkit {
-    max-width: 600px;
+.sk-a {
+    text-decoration: none;
 }
-.sk-outer {
-    margin: 0 auto;
-    width: 100%;
-    max-width: 600px;
-    border-spacing: 0;
-
-    background-image: url('https://drive.google.com/uc?export=view&id=1dQKwKeUTLuZFi64MaGJNrTLPuv_t4wY1');
-    background-repeat: repeat;
-    background-size: 100%;
 
-    padding-bottom: 50px;
-}
-.sk-table {
-    border-spacing: 0;
-    width: 100%;
-}
-.sk-td {
-    padding: 0;
-    vertical-align: top;
+.sk-right-pad{
+    padding-right: 25px;
 }
 
-.sk-a {
-    text-decoration: none;
+.sk-left-pad{
+    padding-left: 25px;
 }
 
 .sk-entry{
     padding-top: 25px;
 }
 
-.sk-desc-pad{
+.sk-left-desc-pad{
     padding-right: 110px;
     padding-top: 8px;
 }
 
+.sk-right-desc-pad{
+    padding-left: 110px;
+    padding-top: 8px;
+}
+
 .sk-detail-pad{
     padding-top: 8px;
 }
 
+.sk-right{
+    text-align: -webkit-right;
```

Comment:
does webkit only work for apple mail? or for gmail mobile / gmail desktop as well

Reply:
It works for all of them. Without webkit, nothing was aligning for apple mail.